tests/pass/intptrcast.rs

```diff
@@ -1,4 +1,5 @@
 //@compile-flags: -Zmiri-permissive-provenance
+//@only-miri: FIXME: CI found a miscompile: `assert_eq!(2, 2)` failed in `should_be_ub`
```

Comment:
😆 apparently LLVM thinks it's UB indeed, just as the comment on the test function suggests

Reply:
Hehe, nice :D

Reply:
Ah should use cfg(miri) here, too